pkg/controller/storagecluster/storagecluster_controller_test.go

```diff
@@ -125,7 +125,7 @@ var mockNodeList = &corev1.NodeList{
 		Kind: "NodeList",
 	},
 	Items: []corev1.Node{
-		corev1.Node{
+		{
```

Comment:
All changes like this are unrelated formatting changes, please move to their own commit.